common/web/keyman-version/build.sh

```diff
@@ -52,7 +52,7 @@ if builder_has_action build; then
   # Generate index.ts
   echo "
     // Generated by common/web/keyman-version/build.sh
-    namespace KEYMAN_VERSION {
+    namespace com.keyman.KEYMAN_VERSION {
```

Comment:
I think we need to talk about namespaces before we start extending the problem. `com` is a bad global and we should be trying to eliminate it, not make it worse or extend it into other projects.

Reply:
Right?  https://github.com/keymanapp/keyman/pull/6539#discussion_r874329854

Reply:
Yeah, but now you are introducing `com` into Keyman Developer Server -- which I don't believe you have fixed in this PR either?

Reply:
Where is it being introduced to the server?  Assuming it works via modules, it shouldn't land there.  Well, assuming my understanding of how modules operate is correct.  It won't survive the trip through `module.exports`.

Reply:
Ah... based on what I'm seeing in the build, it looks like the lexical model compiler parts are using reference-linking rather than module-style inclusion for the `KEYMAN_VERSION` structure.  That'd be where, then.